nexus-repository-helm/src/main/java/org/sonatype/repository/helm/internal/restore/HelmRestoreFacetImpl.java

```diff
@@ -66,10 +67,15 @@ protected void doInit(final Configuration configuration) throws Exception {
   @TransactionalTouchBlob
   public void restore(final AssetBlob assetBlob, final String path) throws IOException {
     StorageTx tx = UnitOfWork.currentTx();
-    HelmAttributes attributes = helmAttributeParser.getAttributesFromInputStream(assetBlob.getBlob().getInputStream());
-    Asset asset = componentRequired(path)
-        ? helmFacet.findOrCreateAsset(tx, path, AssetKind.HELM_PACKAGE, attributes)
-        : helmFacet.findOrCreateAsset(tx, path, AssetKind.HELM_INDEX, attributes);
```

Comment:
Missing `HELM_INDEX` ?

Reply:
HelmAttributes should be empty when HELM_INDEX